test/fixtures.py

```diff
@@ -0,0 +1,51 @@
+"""This file contains all fixtures that are needed by multiple test files."""
+from shlex import split
+from subprocess import run
+from textwrap import dedent
+
+from pytest import fixture
+
+from docker_cli.utils import generate_codename
+
+from .utils import determine_scope, remove_docker_image
+
+
+@fixture(scope=determine_scope)
+def image_tag():
+    """
+    Returns an image tag
+
+    Returns
+    ------
+    str
+        An image tag
+    """
+    tag = f"isce3_pytest_temp_{generate_codename(k=10)}"
+    yield tag
```

Comment:
No need to `yield` here, I think, since there's no teardown step after the `yield` line.

```suggestion
    return f"isce3_pytest_temp_{generate_codename()}"
```